vuepress/docs/next/tutorials/curate/ecr-private-images.md

```diff
@@ -17,7 +17,7 @@ The first step demonstrates how to create a Secret for Docker Hub but please see
 **1. Create the secret**
 Supply the following parameters:
 * the name of the new Secret, e.g. `my-docker-secret`.
-* the URL to your registry server. For Docker Hub this is currently <https://index.docker.io/v1/>
+* the URL to your registry server. For Docker Hub this is currently <https://hub.docker.com/_/registry>
```

Comment:
@nshaw, is this new link correct?

Reply:
Let's keep the original value. My docker info still reports that URL to identify the registry.